packages/ember-engines/UPGRADE.md

```diff
@@ -20,5 +20,7 @@
     > migrate to using router service from ember-engines-router-service
 
 5. drop support for query param only router service transitions until it's resolved upstream https://github.com/villander/ember-engines-router-service/issues/74
+    > example for unsupported syntax ❌ `this.router.transitionTo({ queryParams: { page: 1 }});` without passing a route name  
```

Comment:
Great documentation to improve ergonomic usage. Thank you @void-mAlex

Reply:
@villander is that issue something you'd actively try to fix? don't think it's something widely used, but obviously people do use it since it was reported.

Reply:
@villander did you had a chance to look into it? IMO would be great to fix that and at least https://github.com/villander/ember-engines-router-service/pull/95 before we recommend ppl using that addon OOTB